radio/src/lua/api_lcd.cpp

```diff
@@ -415,7 +415,7 @@ static int luaLcdDrawFilledRectangle(lua_State *L)
   return 0;
 }
 
-#if !defined(COLORLCD)
+
 /*luadoc
 @function lcd.drawGauge(x, y, w, h, fill, maxfill)
```

Comment:
I thought about it, but https://dsbeach.gitbooks.io/opentx-lua-reference-guide/content/lcd/drawGauge.html does not mention that options, so should I add it and we change the doc ?